lib/slather/coverage_service/hardcover.rb

```diff
@@ -0,0 +1,89 @@
+module Slather
+  module CoverageService
+    module Hardcover
+
+      def yml_filename
+        '.hardcover.yml'
```

Comment:
I'm thinking maybe we can just add this values to the `.slather.yml` so they only need one? Like we do for the ci_access_token https://github.com/xing/slather/blob/new_hardcover/lib/slather/project.rb#L107 (on a side note, it looks like I could probably DRY up those `_from_yml` methods to have less duplicate code)